src/components/styles/SearchBar.styled.js

```diff
@@ -2,11 +2,11 @@ import styled from "styled-components";
 
 export const StyledSearchBar = styled.div`
   input {
-    margin-left: 7em;
-    margin-bottom: 0.5em;
-    margin-top: 1em;
     border: solid 0.15em limegreen;
     border-radius: 0.5em;
     color: limegreen;
+    margin-bottom: 0.5em;
+    margin-left: 7em;
+    margin-top: 1em;
```

Comment:
Great work reformatting the styled components structure to account for importance of rules